### ISSUE TITLE
Why single choice checkboxes does not return any value in R shiny.sementic package?

### ISSUE THREAD

**Issue by DamienGeorges** [Appsilon/shiny.semantic#385]
Hi,

I have to build an app using ```shiny``` (and ideally ```shiny.sementic``` for it nice and modern design) where my checkboxes choices is updated dynamically (using reactive). But when I have a single choice, my checkbox output is contently ```NULL``` unrespective the box is checked or not. 
Do you know why?

Please see the example bellow:

```
# Checkbox
library(shiny)
library(shiny.semantic)

ui <- function() {
  shinyUI(
    semanticPage(
      title = "Checkbox example",
      h1("Checkboxes with multiple choices"),
      multiple_checkbox("checkboxes_01", "Select Letters", LETTERS[1:6], selected = "A"),
      p("Selected letters:"),
      textOutput("selected_letters"),
      tags$br(),
      h1("Checkboxes with single choices"),
      multiple_checkbox("checkboxes_02", "Select Letter", LETTERS[1], selected = "A"),
      p("Selected letter:"),
      textOutput("selected_letter")
    )
  )
}

server <- shinyServer(function(input, output) {
  output$selected_letters <- renderText(paste(input$checkboxes_01, collapse = ", "))
  output$selected_letter <- renderText(paste(input$checkboxes_02, collapse = ", "))
})

shinyApp(ui = ui(), server = server)
```

![Screenshot from 2021-10-11 12-16-48](https://user-images.githubusercontent.com/9448889/136776269-dcafef7d-02bf-494e-aac9-a634418f191a.png)

How can I make ```input$checkboxes_02``` return ```A``` when the box is checked in this example?

Note: everithing works right when I am having 2 or more choices (see ``checkboxes_01``` in the example)

Thanks!

**Comment by DamienGeorges** [comment]
Well it seems to be an error linked with the dev version of ```shiny.semantic```available on github (0.4.3).  Re-installing cran version (0.4.2) solve the issue. Thanks for putting me on the right direction! And maybe something to have a keep in mind for the future release of the package.

All the best,
Damien

**Comment by DamienGeorges** [comment]
Hi, 

Sorry to reopen this topic but the bugfix proporsed by @ashbaldry (Thanks!) has not been added to the `develop` (nor `master`) version of `shiny.semantic`. Any chance to have it done in a near future?

All the best,
Damien
